rundeckapp/grails-app/controllers/rundeck/controllers/ScheduledExecutionController.groovy

```diff
@@ -1112,6 +1194,64 @@ class ScheduledExecutionController  extends ControllerBase{
         }
     }
 
+    @Post(uri = '/job/{id}/execution/disable')
+    @Operation(
+        method = 'POST',
+        summary = 'Disable Executions for a Job',
+        description = '''Disable executions for a job. 
+
+Authorization required: `toggle_execution` action for a job.
+
+Since: V14''',
+        tags = ['jobs'],
+
+        parameters = @Parameter(
+            name = "id",
+            description = "Job ID",
+            in = ParameterIn.PATH,
+            required = true,
+            content = @Content(schema = @Schema(implementation = String))
```

Comment:
why not use `schema = @Schema(type = 'string')` as other cases  (`autorest` complains about this too)

Reply:
i believe it is the same result in the yaml, is it not the case? in the result, i see:
```yaml
      - name: id
        in: path
        description: Job ID
        required: true
        content:
          application/json:
            schema:
              type: string
```

Reply:
I got issues with `autorest`, but I think is a `autorest` issue